packages/core/src/payment/strategies/worldpayaccess/worldpayaccess-payment-strategy.ts

```diff
@@ -102,7 +102,8 @@ export default class WorldpayaccessPaymetStrategy extends CreditCardPaymentStrat
             let iframeHidden: HTMLIFrameElement;
             try {
                 iframeHidden = this._createHiddenIframe(error.body);
-            } catch (e) {
+            } catch (error) {
+                console.log(error)
```

Comment:
Can we remove `console.log()` here and line 179?

Reply:
@bc-peng  done, I removed it